pkg/ccl/backupccl/backup_test.go

```diff
@@ -669,7 +669,7 @@ func TestBackupAndRestoreJobDescription(t *testing.T) {
 	asOf1 := strings.TrimPrefix(matches[1], "/full")
 
 	sqlDB.CheckQueryResults(
-		t, "SELECT description FROM [SHOW JOBS] WHERE status != 'failed'",
+		t, "SELECT description FROM [SHOW JOBS] WHERE status != 'failed' AND description LIKE 'BACKUP%'",
```

Comment:
Have you considered not including this new job when SHOW JOB runs
(i.e. require SHOW AUTOMATIC JOBS) in order to display your job (https://github.com/cockroachdb/cockroach/blob/403bbb176c88860208408decdd7d8cb2a10e0534/pkg/sql/delegate/show_jobs.go#L48-L48)?

You might be able to avoid touching so many job implementation tests.  And, tbh, I think it makes sense not to include this job in the show jobs listing (w/out automatic clause)

Reply:
Ack. Made this change. This makes it so I don't have to touch a few tests, but not all of them.